common-util/hooks/useOrbis.jsx

```diff
@@ -5,7 +5,8 @@ import {
   notifyError,
   notifySuccess,
 } from '@autonolas/frontend-library';
-import { mainnet, useAccount } from 'wagmi';
+import { useAccount } from 'wagmi';
+import { mainnet } from 'wagmi/chains';
```

Comment:
thought it should be `'@wagmi/core/chains'` https://wagmi.sh/core/guides/migrate-from-v1-to-v2#chains

but if that works then lgtm